pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/Brevkode.kt

```diff
@@ -22,13 +22,21 @@ object Brevkode {
         UT_UNG_UFOER_20_AAR_AUTO,
         UT_VARSEL_SAKSBEHANDLINGSTID_AUTO,
         UT_BARNETILLEGG_ENDRET_AUTO;
+
+        override fun kode(): String = this.name
     }
 
-    enum class Redigerbar {
+    enum class Redigerbar : Brevkode {
         INFORMASJON_OM_SAKSBEHANDLINGSTID,
         PE_BEKREFTELSE_PAA_FLYKTNINGSTATUS,
         PE_VARSEL_OM_MULIG_AVSLAG,
         UT_ORIENTERING_OM_SAKSBEHANDLINGSTID,
         ;
+
+        override fun kode(): String = this.name
+    }
+
+    interface Brevkode {
```

Comment:
Kan kanskje bare konvertere Brevkode `object` til interface? Grunnen til at vi har `Brevkode`-objektet er egentlig bare for namespacing.